spec/system/admins_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe '管理者', type: :system do
+  it '正しいパスワードのときはログインできること' do
+    create(:administrator, email: 'kuma@test')
```

Comment:
```suggestion
    create(:administrator, email: 'kuma@example.com')
```
思わぬ事故防止！開発時やテスト時に使用するメアドのドメインは example.com に統一しよう
https://qiita.com/jnchito/items/198a2561a36c2c2ef5e3

Reply:
`@test`も安全なの？
っていう気持ちになったし、特に理由なければ`example.com`にしておくのが慣習かなー

Reply:
誤送信のリスク考えてなかったです、、、なるほどそういう観点もあるんですね

テスト書く時はexample.comかsonicgarden.jp（自社ドメイン）に統一するようにします！